api/ssi-bridge/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ssi-bridge",
-  "version": "0.1.8",
+  "version": "0.1.9",
```

Comment:
please remove this I increase the version when the identity update is completed here with the increase to 0.2.0